### PR TITLE
EDUCATOR-5589: Total Grade should always be a percent

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** Total Grade should always be displayed as a percent, as it is a percent.

JIRA: [EDUCATOR-5589](https://openedx.atlassian.net/browse/EDUCATOR-5589)

**What changed?**

- Total Grade should always be displayed as a percent, as it is a percent.
- Previously we had it out of 100, but that makes it look like somehow the course always has an exact total of 100 points. Let's keep it as always a percent to avoid confusion.
- Refactored the label for the "course grade" column out to a constant
- Re-labeled the column "Total Course Grade"

**Developer Checklist**
- [x] Test suites passing
- [x] @sapanathomas523
- [ ] Ben P 
- [x] Bumped version number [package.json](../package.json)

**Testing Instructions**

Change the setting from "Percentage" to "Absolute and back again. The total course grade column should remain a percent.

Screen Shot 2021-04-09 at 1 28 34 PM
Screen Shot 2021-04-09 at 1 28 25 PM

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [x] I've done a visual code review
- [x] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Review by nsprenkle — APPROVED** [review]
Have tested locally. Looks good :shipit:

**Review by jnlapierre — APPROVED** [review]
LGTM

**Comment by jansenk** [comment]
Screen Shot 2021-04-22 at 12 29 38 PM
Screen Shot 2021-04-22 at 12 29 49 PM

**Comment by jansenk** [comment]
Screen Shot 2021-04-22 at 3 46 20 PM
Screen Shot 2021-04-22 at 3 46 07 PM
Screen Shot 2021-04-22 at 3 46 03 PM

**Review by nsprenkle — APPROVED** [review]
Have re-tested locally. Not sure I understand the formatter key change refactoring but the behavior looks good 👍

**Comment by nsprenkle** [comment]
Also, shoutout for showing the ARIA tools output 💯

**Comment by jansenk** [comment]
> Have re-tested locally. Not sure I understand the formatter key change refactoring but the behavior looks good 👍

I wanted to make the `headerLabelReplacements` change in `formatHeadings`, but thought that since we're now doing key checking rather than just accessing by index, it would be safest to have those "key" values pulled out into a constant

File: src/components/Gradebook/GradebookTable.jsx [hunk 1]
```diff
@@ -50,10 +52,10 @@ export class GradebookTable extends React.Component {
     percent: (entries, areGradesFrozen) => entries.map((entry) => {
       const learnerInformation = this.getLearnerInformation(entry);
       const results = {
-        Username: (
+        [USERNAME_HEADING]: (
```
[the remark this answers is not in the document]
Comment (muselesscreator) [reply src/components/Gradebook/GradebookTable.jsx:55]: ++ for removing magic strings

File: src/components/Gradebook/GradebookTable.jsx [hunk 2]
```diff
@@ -120,22 +126,47 @@ export class GradebookTable extends React.Component {
...
-      const emailHeadingLabel = 'Email*';
+      headerLabelReplacements[EMAIL_HEADING] = 'Email*';
 
-      headings = headings.map(heading => ({
-        label: heading,
-        key: heading,
-      }));
+      const totalGradePercentageMessage = 'Total Grade values are always displayed as a percentage.';
+      headerLabelReplacements[TOTAL_COURSE_GRADE_HEADING] = (
+        <div>
```
Comment (muselesscreator) [inline src/components/Gradebook/GradebookTable.jsx:140]: This could be its own component, but is not a big deal

File: src/components/Gradebook/GradebookTable.jsx [hunk 3]
```diff
@@ -120,22 +126,47 @@ export class GradebookTable extends React.Component {
...
+          <OverlayTrigger
+            trigger={['hover', 'focus']}
+            key="left-basic"
+            placement="left"
+            overlay={(<Tooltip id="course-grade-tooltip">{totalGradePercentageMessage}</Tooltip>)}
+          >
+            <div>
+              {TOTAL_COURSE_GRADE_HEADING}
+              <div id="courseGradeTooltipIcon">
+                <FontAwesomeIcon
```
Comment (muselesscreator) [inline src/components/Gradebook/GradebookTable.jsx:150]: Reason we aren't using paragon icon?

**Review by muselesscreator — APPROVED** [review]
LGTM
